cip.go

```diff
@@ -147,17 +169,18 @@ func main() {
 	// the manifest desires a completely blank registry. In practice this would
 	// almost never be the case, so given a fully-parsed manifest with 0 images,
 	// treat it as if -parse-only was implied and exit gracefully.
-	if len(mfest.Images) == 0 {
-		fmt.Println("No images in manifest --- nothing to do.")
-		os.Exit(0)
-	}
+	if len(*snapshotPtr) == 0 {
+		if len(mfest.Images) == 0 {
+			fmt.Println("No images in manifest --- nothing to do.")
```

Comment:
Small nit that doesn't have to be addressed in this PR. I see a lot of `fmt.Print()` around. Do we want to use a logger?